### PR TITLE
[12.0][FIX] product_weight: Adapting test to new constraint

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT28443

Travis is failing because a new constraint was added for phantom bom. This change create the product as consumable to pass the constraint. Here you have the change on Odoo https://github.com/odoo/odoo/blame/3d34d58388ab362bf8e69cc2e75adbdfda3331b2/addons/mrp/models/mrp_bom.py#L85

please @sergio-teruel @pedrobaeza review this

### CODE REVIEW

File: product_weight/tests/test_compute_bom_weight.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2015 Akretion (<http://www.akretion.com>).
+# Copyright 2021 Tecnativa - Carlos Roca
```
Comment (sergio-teruel) [inline product_weight/tests/test_compute_bom_weight.py]: @CarlosRoca13 I think that this is a minor change so you can not add yourself as a file creator..

**Review by sergio-teruel — APPROVED** [review]
Little comment...

**Review by chienandalu — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
@sergio-teruel removed

**Pull request merged**